change/@fluentui-react-e2d6e05f-30c2-46fb-862b-4842bbe7a6d3.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add conditional to enableAriaHiddenOnChildren prop",
```

Since this goes in the changelog for the whole package, it's good to include the component name. Also the prop name is wrong, and probably doesn't need to be explicitly mentioned (it's better to explain the resulting behavior change in this case).
```suggestion
  "comment": "Panel: Only mark siblings as aria-hidden when open",
```